packages/logger/src/middleware/middy.ts

```diff
@@ -99,18 +98,33 @@ const injectLambdaContext = (
     }
   };
 
-  const injectLambdaContextAfterOrOnError = async (): Promise<void> => {
+  const onAfter = async (): Promise<void> => {
     if (isResetStateEnabled) {
       for (const logger of loggers) {
         logger.resetKeys();
       }
     }
+
+    for (const logger of loggers) {
+      logger.clearBuffer();
+    }
+  };
+  const onError = async ({ error }): Promise<void> => {
```

Comment:
Typescript complains this error doesn't have a type. What the type would be. I couldn't see a clear answer in the middy docs.